projects/storefrontlib/src/cms-components/configurator/commons/config-price-summary/config-price-summary.component.ts

```diff
@@ -0,0 +1,33 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import {
+  Configurator,
+  ConfiguratorCommonsService,
+  RoutingService,
+} from '@spartacus/core';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'cx-config-price-summary',
+  templateUrl: './config-price-summary.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfigPriceSummaryComponent implements OnInit {
+  configuration$: Observable<Configurator.Configuration>;
+
+  constructor(
+    private routingService: RoutingService,
+    private configuratorCommonsService: ConfiguratorCommonsService
+  ) {}
+
+  ngOnInit(): void {
+    this.routingService
+      .getRouterState()
+      .subscribe(state => this.initConfigurationPrice(state));
```

Comment:
No subscribe here! (We would need explicit unsubscription). Only work with observables, see config-form e.g.